spec/support/self_service/firm_edit_page.rb

```diff
@@ -47,6 +52,22 @@ def offers_free_initial_meeting?
       !!offers_free_initial_meeting_true.checked?
     end
 
+    def offers_ethical_investing=(choice)
+      send("offers_ethical_investing_#{choice}").set(true)
+    end
+
+    def offers_ethical_investing?
+      !!offers_ethical_investing_true.checked?
```

Comment:
I was following @munckymagik's pattern here — just checked with him and apparently `#checked?` returns something unusual, and we wanted to check against true or false